index.js

```diff
@@ -20,37 +23,35 @@ module.exports = {
         'No Applitools API key found! Set APPLITOOLS_API_KEY with your API key from https://eyes.applitools.com',
       );
     }
-    
-    const applitoolsConfig = {
-      showLogs: true
-    }
-    
-    fs.writeFile(path.resolve(PUBLISH_DIR, '..', 'applitools.config.js'), `module.exports = ${JSON.stringify(applitoolsConfig)}`, 'utf8');
 
     const port = 9919;
     const server = http
       .createServer(ecstatic({ root: `${PUBLISH_DIR}` }))
       .listen(port);
 
-    fs.copy(
```

Comment:
Same here, missing `await` on both `fs.copy` calls. Also I moved `visual-diff.json` into the `visual-diff` directory so we only need a single call.

Another issue is that `PUBLISH_DIR` can be resolved to the repository root (e.g. `publish = "."`). In that case the resolved path would be one level up which is not desired. The code can be simplified by copying to relative path `visual-diff` which will always be resolved currently.